lib/stripe/util.rb

```diff
@@ -213,6 +215,14 @@ def self.check_string_argument!(key)
       key
     end
 
+    def self.validate_params!(params)
```

Comment:
Can we call this something a little more appropriate like `warn_on_opts_in_params`? Lets remove the bang (`!`) too. This is generally used to show that the method may throw an error, which isn't the case here.